settings_vial/settings_vial.py

```diff
@@ -149,3 +149,6 @@ def _load_overrides(self):
 
         for var in vars_to_delete:
             del self._config[var]
+
+    def as_dict(self):
```

Comment:
I think you need to return a copy of the _conf dict here. We don't want 'outsiders' to mutate this dict right ?

Reply:
I think makes sense. Thanks